app/scripts/lib/router.js

```diff
@@ -219,6 +219,7 @@ function (
         return self.showView(view);
       })
       .fail(function (err) {
+        console.error(err);
```

Comment:
That log is actually useful because it provides a full error trace in the console. I can remove it if you want.